### PR TITLE
Refactor ProjectExplorer page-object in the selenium tests

### PR DESCRIPTION
**Pull request by Ohrimenko1988**
### What does this PR do?
Refactoring of methods in the ProjectExplorer page-object

### What issues does this PR fix or reference?
Issue: https://github.com/eclipse/che/issues/7605

### CODE REVIEW

**Comment by Ohrimenko1988** [comment]
ci-test

**Review by tolusha — APPROVED** [review]

**Comment by Ohrimenko1988** [comment]
ci-test

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/ProjectExplorer.java [hunk 1]
```diff
@@ -144,33 +160,39 @@ public ProjectExplorer(
...
   WebElement refreshContextMenuItem;
 
-  @FindBy(xpath = Locators.PROJECT_EXPLORER_TAB_IN_THE_LEFT_PANEL)
+  @FindBy(xpath = PROJECT_EXPLORER_TAB_IN_THE_LEFT_PANEL)
   WebElement projectExplorerTabInTheLeftPanel;
 
+  /**
+   * @param path path to item in Project Explorer
+   * @param timeout timeout in seconds
+   * @return finded WebElement
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/ProjectExplorer.java]: finded > found

Comment (Ohrimenko1988) [reply selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/ProjectExplorer.java]: ok, thx

**Comment by Ohrimenko1988** [comment]
ci-test

**Review by tolusha — APPROVED** [review]

**Comment by Ohrimenko1988** [comment]
ci-test

**Review by SkorikSergey — APPROVED** [review]

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/ProjectExplorer.java [hunk 2]
```diff
@@ -645,6 +639,34 @@ public void revealResourceByOptionsButton() {
...
+  /**
+   * perform the multi-select by Shift key with check selected items
+   *
+   * @param clickItemPath is the path to the selected item
+   * @param selectedItemsPaths is the paths to the each selected items after click with shift button
+   */
+  public void selectMultiFilesByShiftKeyWithCheckMultiselection(
+      String clickItemPath, List<String> selectedItemsPaths) {
+    Actions actions = actionsFactory.createAction(seleniumWebDriver);
+    actions.keyDown(Keys.SHIFT).perform();
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/ProjectExplorer.java]: Let's use static import for simplifying constant references similar to how it had been done in other parts of code.

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/ProjectExplorer.java [hunk 3]
```diff
@@ -568,11 +557,10 @@ public void waitFolderDefinedTypeOfFolderByPath(String pathToFolder, String type
    * @param pathToItem full path to item in the codenvy project explorer
    */
   public void waitRemoveItemsByPath(String pathToItem) {
-    String locator = "//div[@path='/" + pathToItem + "']/div";
-    new WebDriverWait(seleniumWebDriver, 10)
-        .until(
-            ExpectedConditions.not(
-                ExpectedConditions.presenceOfAllElementsLocatedBy(By.xpath(locator))));
+    loadPageTimeout.until(
+        ExpectedConditions.not(
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/ProjectExplorer.java]: Let's use static import for simplifying static method references similar to how it had been done in other parts of code.

**Comment by Ohrimenko1988** [comment]
ci-test

**Comment by dmytro-ndp** [comment]
@Ohrimenko1988: could you, please, take a look at the weird failures of the next tests from the latest ci-test execution:
- [org.eclipse.che.selenium.workspaces.ProjectStateAfterRefreshTest#checkRestoreStateOfProjectAfterRefreshTest](http://tests.codenvycorp.com/selenium_test_report_data/254/failsafe-reports/emailable-report.html#org.eclipse.che.selenium.workspaces.ProjectStateAfterRefreshTest#checkRestoreStateOfProjectAfterRefreshTest) 
- [org.eclipse.che.selenium.factory.CheckFactoryWithSparseCheckoutTest#acceptFactoryWithSparseCheckout](http://tests.codenvycorp.com/selenium_test_report_data/254/failsafe-reports/emailable-report.html#org.eclipse.che.selenium.factory.CheckFactoryWithSparseCheckoutTest#acceptFactoryWithSparseCheckout)

They have been stable during the last 6 runs at the https://ci.codenvycorp.com/view/qa/job/che-integration-tests-che6/

**Comment by Ohrimenko1988** [comment]
@dmytro-ndp 
for 
[org.eclipse.che.selenium.workspaces.ProjectStateAfterRefreshTest#checkRestoreStateOfProjectAfterRefreshTest](http://tests.codenvycorp.com/selenium_test_report_data/254/failsafe-reports/emailable-report.html#org.eclipse.che.selenium.workspaces.ProjectStateAfterRefreshTest#checkRestoreStateOfProjectAfterRefreshTest) 
it looks like in this issue the similar problem:
https://github.com/eclipse/che/issues/7551

for
[org.eclipse.che.selenium.factory.CheckFactoryWithSparseCheckoutTest#acceptFactoryWithSparseCheckout](http://tests.codenvycorp.com/selenium_test_report_data/254/failsafe-reports/emailable-report.html#org.eclipse.che.selenium.factory.CheckFactoryWithSparseCheckoutTest#acceptFactoryWithSparseCheckout)
issue 
https://github.com/eclipse/che/issues/7959

**Review by dmytro-ndp — APPROVED** [review]

**Pull request merged**